scripts/e2e.ts

```diff
@@ -89,18 +89,27 @@ class E2ETestRunner extends events.EventEmitter {
    * Start the renderer process.
    */
   startRendererProcess(): Promise<void> {
-    this.#rendererProcess = this.spawn(
-      'Renderer process',
-      'yarn',
-      'run',
-      'build:ui',
-      '--hostname',
-      'localhost',
-      '--port',
-      this.rendererPort.toString(),
-    );
-
-    return Promise.resolve();
+    return new Promise((resolve, reject) => {
+      this.#rendererProcess = this.spawn(
```

Comment:
Given that we wait for the process to finish, wouldn't using `return buildUtils.spawn(…)` be clearer? That would mean you don't need to watch for the exit signal manually (it handles that).